paimon-spark/paimon-spark-common/src/main/scala/org/apache/paimon/spark/PaimonBaseScanBuilder.scala

```diff
@@ -38,12 +38,12 @@ abstract class PaimonBaseScanBuilder(table: Table)
 
   protected var pushed: Array[(Filter, Predicate)] = Array.empty
```

Comment:
Can we rename this to `pushedFilters`?

Reply:
`pushedFilters` conflicts with Spark method name, rename it to `pushedPredicates`